editor/block-settings-menu/index.js

```diff
@@ -50,6 +51,7 @@ function BlockSettingsMenu( { uids, onSelect } ) {
 				<div className="editor-block-settings-menu__content">
 					<BlockInspectorButton onClick={ onClose } />
 					{ count === 1 && <BlockModeToggle uid={ uids[ 0 ] } onToggle={ onClose } /> }
+					{ count === 1 && <UnknownConverter uid={ uids[ 0 ] } /> }
```

Comment:
Does it make sense to include this in BlockToolbar (for mobile controls)?

Reply:
I'm not entirely sure what you mean.

Reply:
The block settings menu is hidden on mobile and these controls are shown in the block toolbar instead here lob/master/editor/block-toolbar/index.js#L143 Thinking we could do the same with this control.